en16931/src/Class1.cs

```diff
@@ -4,6 +4,11 @@ public class Class1
 {
     public static bool test()
     {
-        return true;
+        return false;
+    }
+
+    public static void validateXRechnungFromFile(string filepath)
```

Comment:
Nit: I believe (static) methods are written in `PascalCase` and not `camelCase` by convention.

I'd also generalise the name further for now, as we _might_ want to be able to parse other standards besides XRechnung, like Factur-X for example, through the same interface:

```suggestion
    public static void ValidateFromFile(string filepath)
```